### DISCUSSION TITLE
Installer does not launch

### DISCUSSION THREAD

**Discussion by Neo166** [mafiosnik777/enhancr#25] [General]
No matter even if i use admin permissions and i try to launch.. the installer would just not open. It would for a fraction of a second run and then immediately quit. I just have the ordinary windows defender and there are no alerts.

**Comment by mafiosnik777** [comment]
No idea, try installing vcredist AIO: https://www.techpowerup.com/download/visual-c-redistributable-runtime-package-all-in-one/

**Comment by 78Alpha** [comment]
Might need to launch it with something like --no-sandbox

This is what I had to do to install it. However, it was with the executable inside the installer. I had to extract the installer and run the Exe inside with --no-sandbox

**Reply by Neo166** [reply]
Thanks surprisingly it started working fine.
